decidim-admin/app/commands/decidim/admin/update_participatory_process.rb

```diff
@@ -21,17 +21,25 @@ def initialize(participatory_process, form)
       # Returns nothing.
       def call
         return broadcast(:invalid) if form.invalid?
-
         update_participatory_process
-        broadcast(:ok)
+
+        if @participatory_process.valid?
+          broadcast(:ok, @participatory_process)
+        else
+          form.errors.add(:hero_image, @participatory_process.errors[:hero_image]) if @participatory_process.errors.include? :hero_image
```

Comment:
As talked offline: The thing is that we would need ImageMagick to do it, since we're inspecting the dimensions of an image. We'll probably need to find a way for commands to inject model errors to forms in an elegant way.